src/components/BuildList.tsx

```diff
@@ -187,8 +188,18 @@ const BuildList: FunctionComponent = () => {
             }?`}</Typography>
           }
           onSubmit={() => {
+            let indexOfBuildDeleted = buildList.findIndex((e) => e.id === menuBuild.id);
```

Comment:
in case deleted build was not selected we loose focus and select first item in list

Reply:
Very good find. Thanks for pointing it out. I have fixed this.